src/sections/SponsorsSection.js

```diff
@@ -1,82 +1,63 @@
-        </a>
-        !
       </h3>
       <div className={styles.sponsorsSection}>
-        <TabletComponent>
           <div>
             <div className={styles.sponsors}>
               <Sponsor
-                sponsor="google.png"
-                url="https://careers.google.com/"
+                sponsor="Peak6.jpg"
+                url="https://peak6.com/careers/"
                 tier="gold"
               />
               <Sponsor
-                sponsor="crowdstrike.png"
-                url="https://www.crowdstrike.com/careers/"
+                sponsor="Motorola.jpg"
+                url="https://www.motorolasolutions.com/en_us/about/careers.html"
                 tier="gold"
               />
               <Sponsor
-                sponsor="cmegroup.jpg"
-                url="https://www.cmegroup.com/careers.html"
-                tier="gold"
-              />
-              <Sponsor
-                sponsor="twoSigma.png"
-                url="https://www.twosigma.com/careers/"
-                tier="gold"
-              />
-              <Sponsor
-                sponsor="drw.png"
-                url="https://drw.com/work-at-drw"
-                tier="gold"
+                sponsor="IMCTrading.png"
+                url="https://www.imc.com/us/search-careers"
+                tier="bronze"
               />
               <Sponsor
-                sponsor="omc.png"
+                sponsor="OldMission.png"
                 url="https://www.oldmissioncapital.com/careers/"
                 tier="gold"
               />
-              <Sponsor
-                sponsor="hrt.png"
-                url="https://www.hudsonrivertrading.com/careers//"
-                tier="gold"
-              />
             </div>
             <div className={styles.sponsors}>
-              <Sponsor
-                sponsor="bloomberg.png"
-                url="https://www.bloomberg.com/company/careers/early-career/"
+            <Sponsor
+                sponsor="drw.png"
+                url="https://drw.com/work-at-drw"
                 tier="bronze"
               />
-              <Sponsor
-                sponsor="activeCampaign.png"
-                url="https://www.activecampaign.com/about/careers"
+            <Sponsor
+                sponsor="hrt2.png"
+                url="https://www.hudsonrivertrading.com/careers/"
                 tier="bronze"
               />
               <Sponsor
-                sponsor="citadel.png"
-                url="https://www.citadel.com/careers/"
+                sponsor="CMEGroup1.png"
+                url="https://www.cmegroup.com/careers.html"
                 tier="bronze"
               />
             </div>
           </div>
-        </TabletComponent>
+          If you are interested in sponsoring us, please email{' '}
+        <a
+          className={styles.sponsorEmail}
+          href="mailto:contact@illinoiswcs.org"
+        >
+          contact@illinoiswcs.org!
```

Comment:
i think this is my mistake, but it should be illinoiswcs@gmail.com instead of contact@illinoiswcs.org

Reply:
actually don't change this for right now. but i might have you come back and change it if i get confirmation abt this.